tips/tests/tips_views_tests.py

```diff
@@ -7,7 +7,9 @@
 from tips.forms import TipForm
 from tips.views import board, delete_tip
 from django.template.loader import get_template
-from tips.constants import TIPS_PAGE_ROUTE, ADD_TIP_PAGE_ROUTE, EDIT_TIP_PAGE_ROUTE
+from tips.constants import TIPS_PAGE_ROUTE, ADD_TIP_PAGE_ROUTE, EDIT_TIP_PAGE_ROUTE, READ_MORE_PAGE_ROUTE, \
```

Comment:
Is the '\' at the end necessary?

Reply:
The '\' is meant to continue the code in new line

Reply:
ok 👍